### PR TITLE
Luca-AlexanerThomas.AutoHome-Autostart version 1.0.0

### PR DESCRIPTION
**Pull request by luca-alexander-thomas**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25870)

### CODE REVIEW

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
`Installer failed with exit code: 2`

**Comment by luca-alexander-thomas** [comment]
What i have to do here?

**Comment by luca-alexander-thomas** [comment]
I need Help

**Comment by ItzLevvie** [comment]
image

Your installer doesn't seem to open on my machine from `%TEMP%\WinGet`. I recommend testing your package in a virtual machine to make sure that it works and returns "Successfully installed." when installing it through WinGet.

**Comment by luca-alexander-thomas** [comment]
Is there anything that i can do to restart the Checks?

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
You can close and reopen the PR to rerun the pipelinex.

**Review by luca-alexander-thomas — COMMENTED** [review]
edit the installer URL

File: manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanerThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: nullsoft
```
Comment (ItzLevvie) [inline manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.installer.yaml:8]:
```suggestion
  InstallerType: msi
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.3.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Luca-AlexanerThomas.AutoHome-Autostart
+PackageVersion: 1.0.0
+PackageLocale: de-DE
+Publisher: Luca-Alexander Thomas
+PublisherUrl: https://lucathomas.de
+PublisherSupportUrl: https://lucathomas.de/kontakt
+PrivacyUrl: https://lucathomas.de/datenschutzerklaerung/
```
Comment (ItzLevvie) [inline manifests/l/Luca-AlexanerThomas/AutoHome-Autostart/1.0.0/Luca-AlexanerThomas.AutoHome-Autostart.locale.de-DE.yaml:10]:
```suggestion
#PublisherUrl: https://lucathomas.de
#PublisherSupportUrl: https://lucathomas.de/kontakt
#PrivacyUrl: https://lucathomas.de/datenschutzerklaerung/
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by luca-alexander-thomas — COMMENTED** [review]
edit all Manifests

**Pull request closed without merging**